kin-backup-and-restore/kin-backup-and-restore/src/androidTest/java/kin/backupandrestore/qr/QRFileUriHandlerImplTest.java

```diff
@@ -32,7 +32,8 @@ public void saveFile_Success() throws Exception {
 		Bitmap bitmap = TestUtils.loadBitmapFromResource(this.getClass(), "qr_test.png");
 		Uri uri = fileUriHandler.saveFile(bitmap);
 		assertThat(uri, notNullValue());
-		assertThat(uri.toString(), equalTo("content://kin.recovery.test.KinRecoveryFileProvider/qr_codes/backup_qr.png"));
+		assertThat(uri.toString(),
+			equalTo("content://kin.backupandrestore.test.KinRecoveryFileProvider/qr_codes/backup_qr.png"));
```

Comment:
Should probably be a constant variable for readability

Reply:
it is only in the test file but will do it